### DISCUSSION TITLE
User feedback

### DISCUSSION THREAD

**Discussion by ingo-m** [glassflow/glassflow-python-sdk#8] [General]
@ashish-bagri Congratulations for launching the MVP 🚀

I like the approach of having a simple and elegant CLI ✨

Here's some feedback after trying it out for the first time. I went through the Quickstart tutorial, but I used my own data file to simulate a log stream.

I'm using Ubuntu 22.04.4 LTS, Python 3.10.14.

- I installed glassflow via the bash command given in the documentation (`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/glassflow/cli/master/install.sh)"`). The glassflow binary is created in my home directory, but I cannot run `glassflow --version`, I need to run `/home/john/glassflow --version`.

    ![glassflow_feedback_01](https://github.com/glassflow/glassflow-python-sdk/assets/10960892/4e0e0314-810e-469c-975f-0438ee859102)

    ![glassflow_feedback_02](https://github.com/glassflow/glassflow-python-sdk/assets/10960892/cbc4c1d0-6f95-460f-a1f6-d6f6f4664ba9)

- When I created my first pipeline, I found the naming a bit confusing. The `.env` file requires the following variables: `PIPELINE_ID`, `SPACE_ID`, `PIPELINE_ACCESS_TOKEN`, but when creating the pipeline with the CLI, it mentions the "pipline name", "space name" and "Push token". I suppose `PIPELINE_ID` = pipeline name, `SPACE_ID` = space name, and `PIPELINE_ACCESS_TOKEN` = push token. 🤔

- I can see my pipeline with `./glassflow pipeline list`, but `/glassflow pipeline logs ingotestlogfile --space main` says no logs are found, even though I think I published a few thousand events. When I run the `consumer.py` script, there is no error, but no events are consumed. I'm not sure, could be that I made a mistake. I'm not sure whether the issue is with publishing, transforming, or consuming. But there was no error message while I was publishing (i.e. `pipeline_client.publish()` returned `status_code` 200), so I assume the error is at the transformation or consuming stage.

    ![glassflow_feedback_03](https://github.com/glassflow/glassflow-python-sdk/assets/10960892/23698e6a-4c86-4b9b-9470-289b3ca33dc2)

- Finally a minor thing I noticed in the web UI: When I refresh the page I have to log in again.

    https://github.com/glassflow/glassflow-python-sdk/assets/10960892/f657c6f3-3509-4a73-8984-6c882dbbf117

**Comment by Boburmirzo** [comment]
@ingo-m Hi Ingo,

Thanks for using GlassFlow and providing very valuable feedback!

We will take quick actions and update the doc accordingly:) 

Let us know if you find any other issues. I will be happy to assist.

**Reply by Boburmirzo** [reply]
@ingo-m According to your input, I have just updated the following docs to reflect changes:

Addtional guideline to move the GlassFlow CLI so it's accessible system-wide on Linux:

https://learn.glassflow.dev/docs/get-started/glassflow-cli#install-from-the-release-package

When you create a space or pipeline by providing a name, it will become a uniquely identifier ID for the space or pipeline. The space or pipeline can be accessed by its ID. Read more about the [resource naming](https://learn.glassflow.dev/docs/concepts/pipeline-configuration#resource_names).

https://learn.glassflow.dev/docs/get-started/quickstart#step-5-create-your-pipeline

Thanks again for your contribution!

**Comment by alexandertrefz** [comment]
Hey @ingo-m,
We have deployed a new Version of the Webapp, which hopefully resolves the Authentication issue you faced, can you tell me if it is fixed for you now?

**Reply by ingo-m** [reply]
Hi @alexandertrefz,

Sorry it took me some time to get back to this. I tried again a few days ago and put quite a lot of events through the pipeline, it worked as expected. ✅

**Reply by Boburmirzo** [reply]
Thanks @ingo-m !
